### DISCUSSION TITLE
app/How-to-use-Hiddify-app/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4462] [Content Help کمک به اصلاح محتوا]
# app/How-to-use-Hiddify-app/

The proxy manager platform.

https://hiddify.com/app/How-to-use-Hiddify-app/

**Comment by sailpwnz** [comment]
How do I run Hiddify on Linux Ububtu 24.04 and Debian 12.7 if I installed via dpkg -i?

**Reply by sailpwnz** [reply]
Sorry, run Hiddify GUI with administrator rights, for TUN mode

**Comment by nexus181** [comment]
Why are the remaining days and data usage not displayed?
![image](https://github.com/user-attachments/assets/3a953c00-f554-4997-b057-748132115e73)

**Comment by nazim33** [comment]
Unexpected Error

listen: listen tcp 127.0.0.1:8964: bind: An attempt was made to access asocket in a way forbidden by its access permissions.  why it shows?

**Comment by rectomiriko** [comment]
what is the difference between 'vpn' and 'vpn service' modes?

**Comment by JeremyYuri** [comment]
Please adapt to the HarmonyNEXT, pls!
